stdlib/3.4/asyncio/queues.pyi

```diff
@@ -32,15 +33,19 @@ class Queue(Generic[T]):
     @coroutine
     def get(self) -> Future[T]: ...
     def get_nowait(self) -> T: ...
+    if sys.version_info >= (3, 4, 4):
```

Comment:
In other PRs with the same problem we assumed that for changes introduced in *any* "3.4.x" release we're going to use  `>= (3, 4)`. This way the type checker stays silent if the user is running a version that is too old. A false negative is okay. If we used `>= (3, 5)` instead, the user of, say, 3.4.4 would get false positives.